Kinvey/Kinvey/Client.swift

```diff
@@ -96,7 +96,7 @@ open class Client: Credential {
     open var clientAppVersion: String?
     
     /// Custom request properties for this client instance.
-    open var customRequestProperties: [String : String] = [:]
+    open var customRequestProperties: [String : Any]?
```

Comment:
good question... I'm going to check with the requester of this feature if make sense to keep a global version